website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,179 @@
+
+For example, the following command will run the `export_1` and `export_2` and doesn't work with the `--alias` or `--export_as` flags:
+
+```bash
+dbt sl export --select export_1,export2
+```
+
+<details>
+<summary>Overriding Export configurations</summary>
+
+The `--select` flag is mainly used to include or exclude specific exports. If you need to change these settings, you can use the following flags to override Export configurations:
+
+- `--export-as` &mdash; Defines the materialization type (table or view) for the Export. This creates a new export with its own settings, useful for testing in development.
+- `--schema` &mdash;  Specifies the schema to be used for the  materialized table or view.
+- `--alias` &mdash; Assigns a custom alias to the materialized table or view, overriding the default Export name.
+
+Note that `--select` flag can't be used with `alias` or `schema`.
+
+For example, you can use the following command to create a new Export named `new_export` as a table:
+
+```bash
+dbt sl export --saved-query sq_number1 --export-as table --alias new_export
+```
+</details>
+
+### Exports in Production
+
+You can run an Export against your production data by adding the `--include-saved-query` flag to the `dbt build` **Commands** box, in your deployment **Execution Settings**. Running `dbt build --include-saved-query` runs the equivalent of `dbt sl export --saved-query sq_name` in the dbt Cloud Job scheduler.
+
+When you run a build job, any saved queries downstream of the dbt models in that job will run as well. To make sure your Export data is up-to-date, run the Export as a downstream step (after the model). The steps to create an Export are:
+
+1. Create a [deploy job](/docs/deploy/deploy-jobs) in dbt Cloud.
+2. Add the `--include-saved-query` to the `dbt build` command in your **Execution Settings** to re-run any Export that needs to be refreshed after a model is build. 
+   - You can use the [selector syntax](/reference/node-selection/syntax) `--select` to only run the exports downstream of a particular model.
+3. After dbt finishes building the models, the MetricFlow Server processes the exports, compiles the necessary SQL, and executes this SQL against your data platform.
+4. Review the exports execution details in the jobs logs and confirm the Export was run successfully. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the job logs.
+5. Your data is now available in the data platform for querying.
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/deploy_exports.jpg" width="90%" title="Adding --include-saved-query to the dbt build command in your job execution settings." />
+
+You can use the selector syntax `--select` or `-s` to specify a particular dbt model to run in your build command in order to only run the exports downstream of that model. As an example, the following command will run any saved queries that are downstream of the `orders` semantic model:
```

Comment:
consider moving this info (along with example) to step 2's sub-bullet instead, where `--select` is first mentioned 

by chunking together, it won't feel like the sentence is being repeated.